botx/client/chats_api/exceptions.py

```diff
@@ -1,6 +1,14 @@
 from botx.client.exceptions.base import BaseClientException
 
 
+class ChatMembersNotModifiableError(BaseClientException):
+    """Can't edit a list of personal chat administrators."""
+
+
+class AdministratorsNotChangedError(BaseClientException):
```

Comment:
`AdminsNotChangedError`? We can use shortening + botx error is `admins_not_changed`